tox/config.py

```diff
@@ -910,6 +910,16 @@ def __init__(self, name, indexserver=None):
         self.name = name
         self.indexserver = indexserver
 
+    @property
+    def digest(self):
+        fname = str(self.name)
+        if fname.startswith('-r'):
+            fname = fname[2:]
```

Comment:
There may be optional whitespace: `-r requirements.txt`

Reply:
Actually, there can be at the moment, but it won't work as expected: it would search for requirements.txt (note space). See PR #669.

You should check for --requirement as well, in which case you will have to strip an =.

A requirements file itself can depend on a requirements file (e.g. dev requirements on install requirements). I guess that is not being checked here?